compile-android.sh

```diff
@@ -8,13 +8,19 @@ if [ -z "$ANDROID_NDK_HOME" ]; then
     exit 1
 fi
 
-# Ring won't build in the below target for whatever reason
-# 'arm armv7-linux-androideabi' \
+if [ $RELEASE ]; then
+    echo "Building in release mode... (slow)";
+else
+    echo "Building in debug mode... (fast)"
+fi
 
 ORIG_PATH="$PATH"
 
+# Ring won't build in the below target for whatever reason
+# 'arm armv7-linux-androideabi' \
 for archtargetstr in \
     'x86 i686-linux-android' \
```

Comment:
Just remove 32-bit support completely, it just slows down testing time. I have a 10 year old laptop and even that has a 64-bit CPU. The Core 2 series were the first mainstream 64-bit Intel CPUs and they have been available for 13 years now!

Reply:
I don't know why, but Google seems to suggest supporting 32-bit x86 is normal: https://developer.android.com/distribute/best-practices/develop/64-bit

Could check the device support matrix and see if there actually are any 32-bit x86 devices out there that don't support x86_64, but until then I'll keep it.

Reply:
Might be because of those mobile Intel mobile SoC for smartphones which Intel made one generation of and failed spectacularly in 2014, they were 32-bit. Since then they probably just want x86_64 APKs because of Chromebooks now supporting Android apps.